source/localizable/configuring-ember/debugging.md

```diff
@@ -115,6 +121,8 @@ but a common practice is to call `console.assert` to dump the error to the
 console.
 
 ```app/app.js
+import Ember from 'ember';
+
 Ember.RSVP.on('error', function(error) {
```

Comment:
It's also possible to import RSVP, `import RSVP from 'rsvp';` then use `RSVP` instead of `Ember.RSVP`. See, https://github.com/emberjs/guides/issues/1585 but that is another issue